### DISCUSSION TITLE
about the generatofit in pyextremes

### DISCUSSION THREAD

**Discussion by ghost** [georgebv/pyextremes#17] [Q&A]
Hi!I used Pyextremes to directly select POT to get the extremes data and then get a shape parameter xi . But it is slightly different from the one I used df.loc to get the extremes event and use scipy.generato.fit () to get the shape parameter. What are these differences related to?(r-Timedelta?) My data is daily extreme value data, including 30 years, may I ask whether r=24H still has any influence?Looking forward to your answer.

example code:
data=df['XXX']
#threshold is set to 8 
U=data.loc[data>8]
genpareto.fit(u,loc=8)

**Comment by georgebv** [comment]
@Goddysen I am not sure what is going - please provide code which I could reproduce to answer your question. Currently I do not understand what is `df` (provide code or data to create it) and what things you are comparing.

As for `r="24H"` it has significant effect on your extremes because it is used to de-cluster your exceedances - without declustering you are fitting model noise. I suggest reading more here: https://georgebv.github.io/pyextremes/user-guide/4-peaks-over-threshold/#declustering
